tsconfig.json

```diff
@@ -11,7 +11,9 @@
     "rootDir": "src/",
     "outDir": "lib/cjs/",
     "skipLibCheck": true,
-    "lib": ["dom", "es5", "scripthost", "es2015"]
+    "lib": ["dom", "es5", "scripthost", "es2015"],
+    "jsx": "react",
+    "jsxFactory": "jsxFactory.createElement"
```

Comment:
What do you think of configuring `jsx` and `jsxFactory` here?
Or should we have `test/tsconfig.json` separately instead?